src/hacking/building-for-openharmony.md

```diff
@@ -196,24 +36,30 @@ Otherwise, the environment variables will not be loaded.
 The following command can then be used to compile the servoshell application for a 64-bit ARM device or emulator:
 
 ```commandline
-./mach build --ohos --release [--flavor=harmonyos]
+./mach build --ohos --release --flavor=harmonyos
 ```
 
-In `mach build`, `mach install` and `mach package` commands, `--ohos` is an alias for `--target aarch64-unknown-linux-ohos`.
-To build for an emulator running on an x86-64 host, use `--target x86_64-unknown-linux-ohos`.
-The default `ohos` build / package / install targets OpenHarmony.
-If you want to build for HarmonyOS you can add `--flavor=harmonyos`.
-Please check the [Signing configuration](#signing-configuration) and add a configuration with `"name": "hos"` and `"type": "HarmonyOS""` and the respective signing certificates.
-
-
-## Installing and running on-device
+Now you should have a shared library in /path/to/servo/target/aarch64-unknown-linux-ohos/release/libservoshell.so.
+You can ignore the error about hvigor tool as we will not produce the app bundle on this machine.
```

Comment:
Well, I guess the previous instructions were unclear - but after setting everything up, you can build, package and install all from your Linux workstation. We definitely shouldn't recommend using 2 PCs for building.